util/neo-nrve-eventhandler.py

```diff
@@ -33,7 +33,6 @@
 from neo.Core.Blockchain import Blockchain
 
 from neo.contrib.smartcontract import SmartContract
-from neo.SmartContract.ContractParameter import ContractParameter, ContractParameterType
```

Comment:
It looks like your merge went haywire and removed my changes that are needed here.

Reply:
😇Yep, when I deployed it.. I noticed some odd merges too! Working on this now